go.mod

```diff
@@ -23,13 +23,15 @@ require (
 	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
 	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200321030439-57b580e57e88
 	github.com/operator-framework/operator-marketplace v0.0.0-20190919183128-4ef67b2f50e9
-	github.com/spf13/cobra v1.0.0
+	github.com/sirupsen/logrus v1.7.0 // indirect
+	github.com/spf13/cobra v1.1.1
```

Comment:
The error in unit tests is because of cobra version being increased, the easy fix would be to revert to `v1.0.0` or apply this [fix PR](https://github.com/ricardozanini/kogito-cloud-operator/pull/13) :)

Reply:
I think the update of cobra to v1.1.1 was done due to gogog v0.11.0 on master. But 1.2.x is still with v0.10.0, so we should not need it.

Reply:
@Kaitou786 I have the rights to push to Zanini's repository :D
So I took your commit in above mentioned PR and cherry-picked it to this PR. Let's see ...